src/libraries/System.Threading.Channels/src/System/Threading/Channels/IDebugEnumerator.cs

```diff
@@ -26,4 +26,6 @@ public DebugEnumeratorDebugView(IDebugEnumerable<T> enumerable)
         [DebuggerBrowsable(DebuggerBrowsableState.RootHidden)]
         public T[] Items { get; }
     }
+
+    internal sealed class DebugEnumeratorDebugView<T, TOther>(IDebugEnumerable<T> enumerable) : DebugEnumeratorDebugView<T>(enumerable);
```

Comment:
just for my education: why do we need `TOther` if it's not used by the type?

Reply:
This type is used as a DebuggerTypeProxy. The number of generic type parameters for the proxy needs to match the number for the type it's proxying.

Reply:
> The number of generic type parameters for the proxy needs to match the number for the type it's proxying.

Thanks!